src/components/IconButton/IconButton.jsx

```diff
@@ -77,9 +77,12 @@ const IconButton = forwardRef(
     }, [disabled, disabledReason, tooltipContent, ariaLabel]);
 
     const IconButtonWrapper = wrapperClassName ? "div" : Fragment;
+    const iconButtonWrapperProps = useMemo(() => {
+      return wrapperClassName ? { className: cx(wrapperClassName, styles.wrapper) } : {};
+    }, [wrapperClassName, styles.wrapper]);
```

Comment:
It's looks like we can remove the style modules dependency

Reply:
@hadasfa you are absolutely right